docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,24 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.10.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.10.0) (07-21-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.9.2...v2.10.0)
+
+**Closed issues:**
+
+- Consider @param with type and name [\#421](https://github.com/forcedotcom/sfdx-scanner/issues/421)
```

Comment:
This was closed without any changes. I think it can be removed.

Reply:
ok, will remove it.